core/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -3,10 +3,26 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
+  xmlns:tools="http://schemas.android.com/tools"
   android:fitsSystemWindows="true">
 
-  <include layout="@layout/layout_standard_app_bar" />
+  <com.google.android.material.appbar.AppBarLayout
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintTop_toTopOf="parent"
+    android:theme="@style/Widget.KiwixTheme.AppBar"
```

Comment:
Because this has already been specified in `themes.xml` you should be able to omit this line and the appBar should stay black